Quick.xcodeproj/project.pbxproj

```diff
@@ -2400,6 +2400,7 @@
 				MACOSX_DEPLOYMENT_TARGET = 10.9;
 				METAL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_SWIFT_FLAGS = "$(inherited) -swift-version 3";
```

Comment:
Can you please remove this since this is not needed (https://github.com/Quick/Nimble/pull/437)?